lib/y2_installbase.pm

```diff
@@ -189,6 +189,25 @@ sub move_down {
     check12qtbug if check_var('VERSION', '12');
 }
 
+=head2 process_visible_patterns 
+
+    process_visible_patterns();
+
+Decides to select specific patterns based on setting PATTERNS
+Possible values for PATTERNS
+  PATTERNS=minimal,base
+=cut
+
+sub process_visible_patterns {
+    my ($self) = @_;
+    if (get_required_var('PATTERNS')) {
```

Comment:
`get_required_var` dies if doesn't find the setting, you don't need an `if`
but you don't use it in the function, better follow suggestions below.